ush/set_rrfs_config_general.sh

```diff
@@ -445,7 +445,7 @@ if [[ ${PREDEF_GRID_NAME} == "RRFS_NA_3km" ]] ; then
 
 fi
 
-if [[ ${PREDEF_GRID_NAME} == "RRFS_NA_13km" ]] ; then 
+if [[ ${PREDEF_GRID_NAME} == "GSD_RAP13km" ]] ; then 
```

Comment:
Why do we change from RRFS_NA_13km to GSD_RAP13km?

Reply:
@guoqing-noaa  From the initial setting, the PREDEF_GRID_NAME for 13km NA RRFS runs are GSD_RAP13km instead of RRFS_NA_13km. We could change to RRFS_NA_13km, but I think that would require more work.  These are all the available PREDEF_GRID_NAME: 
"RRFS_CONUS_25km" \
"RRFS_CONUS_13km" \
"RRFS_CONUS_3km" \
"RRFS_CONUS_3km_HRRRIC" \
"RRFS_SUBCONUS_3km" \
"RRFS_AK_13km" \
"RRFS_AK_3km" \
"CONUS_25km_GFDLgrid" \
"CONUS_3km_GFDLgrid" \
"EMC_AK" \
"EMC_HI" \
"EMC_PR" \
"EMC_GU" \
"GSL_HAFSV0.A_25km" \
"GSL_HAFSV0.A_13km" \
"GSL_HAFSV0.A_3km" \
"GSD_HRRR_AK_50km" \
"GSD_RAP13km" \
"RRFS_NA_3km" \

Reply:
So it is a bug in the previous versions and you fixed this bug in this PR, right?

Reply:
@guoqing-noaa That's what I think.

Reply:
@HaidaoLin-NOAA Haidao, I think it does not make sense to keep GSD_RAP13km, we should have RRFS_NA_13km. First of all GSD does not exist anymore. Then RRFS_NA_13km is self-descriptive: it is RRFS physics on North America domain with 13-km resolution.

Reply:
@tanyasmirnova Tanya, Thanks for checking.  Well, I just followed the naming convention, the PREDEF_GRID_NAME for 13km NA RRFS is set to be GSD_RAP13km.  If we use RRFS_NA_13km in here, the "if-fi" section does not work.  We could change to use RRFS_NA_13km for all settings in the future, but I think that will require more fixes.